app/src/main/java/it/unibo/sc/ItemsActivity.kt

```diff
@@ -0,0 +1,47 @@
+package it.unibo.sc
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import com.amplifyframework.auth.AuthException
+import com.amplifyframework.kotlin.core.Amplify
+import it.unibo.sc.databinding.ActivityItemsBinding
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.async
+
+class ItemsActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityItemsBinding
+    private lateinit var deferredLogout: Deferred<Unit>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityItemsBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val button = binding.logoutButton
+        val mainActivityIntent = Intent(this, MainActivity::class.java)
```

Comment:
As in the `MainActivity` could be better if provide a method for changing the activity instead do that in place